docker/compose/kafka-schema-registry.yml

```diff
@@ -47,22 +47,67 @@ services:
             KAFKA_REST_SCHEMA_REGISTRY_URL: http://schema-registry:8081
       ports:
         - "8082:8082"
+  connect:
```

Comment:
Do we need this "connect" for proto handling? Just checking...

The "kafka-cluster.yml" file has this "connect" anyway, if anyone needs it for reference.
So do not delete this file.

Let's remove from "kafka-schema-registry.yml" if not needed ✌️

Reply:
I removed connect from schema registry file